perfkitbenchmarker/providers/profitbricks/__init__.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 """ProfitBricks cloud provider implementation."""
 
-PROFITBRICKS_API = 'https://api.profitbricks.com/cloudapi/v3/'
+PROFITBRICKS_API = 'https://api.profitbricks.com/cloudapi/v4'
```

Comment:
I'm unable to run profitbricks at this time. Which benchmarks have you tested this change on?

Reply:
I tested iperf and ping. The example form README should work.
```
./pkb.py --cloud=ProfitBricks --machine_type=Small --benchmarks=iperf
```